x-pack/plugins/monitoring/public/components/elasticsearch/nodes/nodes.js

```diff
@@ -42,11 +42,11 @@ import { ClusterStatus } from '../cluster_status';
 import { MetricCell, OfflineCell } from './cells';
 
 const tableCellNameStyle = (theme) => css`
-  font-size: ${euiFontSize(theme, 'm')};
+  ${euiFontSize(theme, 'm')}
```

Comment:
Just to point it out in case it's unwanted: There was no `line-height` defined before and util adds `fontSize` and `lineHeight` both (which most of the time should go together).

Reply:
Thanks for the catch! This is wanted. I applied `font-size: ${euiFontSize(theme, 'm').fontSize};` wherever only `font-size` was specified in the SCSS files, otherwise I applied `${euiFontSize(theme, 'm')}` when `@include euiFontSizeM` was specified. I corrected all these cases in: https://github.com/elastic/kibana/pull/204258/commits/06f045abcb2ea175cbc73e0b97c78cdffff1413f making sure to use the correct option.

For this exact case, see: `x-pack/plugins/monitoring/public/components/cluster/listing/listing.scss`:
```scss
.monTableCell__name,
.monTableCell__status {
  @include euiFontSizeM;
}
```

Is this correct?

Reply:
Ah right! You're spot on, I missed that it was `@include euiFontSizeM;` before. So it's definitely correct 👍